mac/Keyman4MacIM/Keyman4MacIM/Privacy/PrivacyWindowController.m

```diff
@@ -0,0 +1,59 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ * 
+ * PrivacyWindowController.m
+ * TestInputMethod2
+ * 
+ * Created by Shawn Schantz on 2022-09-20.
+ * 
+ * Controller for the alert dialog that is displayed at startup by the PrivacyConsent class
+ * when it detects that the application has not been granted sufficient access
+ * (Accessibility or PostEvent, depending on macOS version) to run properly.
+ */
+
+#import "PrivacyWindowController.h"
+
+@interface PrivacyWindowController ()
+@end
+
+@implementation PrivacyWindowController
+
+- (void)windowDidLoad {
+  [super windowDidLoad];
+  
+  NSString *bundleDisplayName = [[[NSBundle mainBundle] localizedInfoDictionary]
+                           objectForKey:@"CFBundleDisplayName"];
+  [self.window setTitle:bundleDisplayName];
+  [_alertText setStringValue:NSLocalizedString(@"privacy-alert-text", nil)];
+  
+  NSFileManager *fileManager  = [NSFileManager defaultManager];
+  NSString *logoPath = [[NSBundle mainBundle] pathForResource:@"keyman-alert-logo" ofType:@"png"];
```

Comment:
Are you able to use `icon_256x256.png` instead of adding `keyman-alert-logo.png`? (Are they the same?)

Reply:
thanks -- I thought it should be available somewhere in the bundle but I didn't see it